windows/mkwinsyscall/mkwinsyscall.go

```diff
@@ -480,6 +486,15 @@ func newFn(s string) (*Fn, error) {
 		return nil, errors.New("Could not extract dll name from \"" + f.src + "\"")
 	}
 	s = trim(s[1:])
+
+	// custom dll extension
+	prefix, body, _, found = extractSection(s, '[', ']')
+	if found {
+		ext := strings.Replace(body, "ext=", "", 1)
+		f.dllExtension = strings.TrimSpace(ext)
+		s = strings.TrimSpace(prefix)
+	}
```

Comment:
- This part of the code would be better to move to just after the code to extract the fail condition so that the code to extract a DLL name and a DLL function name is not separated.
- It would be better to set `f.dllExtension = "dll"` by default in advance rather than verifying it when creating a DLL name
- It would be better to use `trim` instead of `strings.TrimSpace` to follow the current code
- This code doesn't work if whitespace is inserted around `=` like `[ext = drv]`
- Would it be better to return an error if an unexpected key other than `ext` is specified?